redux/actions/currenciesActions.ts

```diff
@@ -18,3 +19,34 @@ export const getUserCurrenciesFromAsync = () => {
     }
   }
 }
+
+export const addCurrencyBySymbol = (valueSearched: string) => {
+  return async (dispatch: Dispatch, getState: () => IState): Promise<boolean> => {
+    const {allCurrencies, userCurrenciesList} = getState().currencies
+
+    const newCrypto: ICurrency | undefined = allCurrencies.find( (currency: ICurrency) => {
```

Comment:
[suggestion] Crypto search should be handled in a single function, not searched and then validated.